docs/Users/Tutorials/Linter_Bears.rst

```diff
@@ -0,0 +1,293 @@
+Linter Bears
+============
+
+Welcome. This tutorial aims to show you how to use the Lint
+class in order to integrate linters in your bears.
+
+Why is This Useful?
+--------------------
+
+A lot of programming languages already have linters implemented, so if your
+project uses a language that does not already have a linter Bear you might
+need to implement it on your own. Don't worry, it's easy!
+
+What do we Need?
+----------------
+
+First of all, we need the linter that we are going to use.
+In this tutorial we will build the HTMLTutorialBear so we need an html
+linter.
+`This <https://github.com/deezer/html-linter>`__ one will do.
+Since it is a python package we can go ahead and install it with
+
+::
+
+	$ pip install html-linter
+
+Writing the Bear
+----------------
+
+Since we are going to use the Lint class we should go ahead and
+import it together with LocalBear (all bears that handle only one file
+inherit from LocalBear). Also we will go ahead and write the class
+head. It should inherit both LocalBear and Lint.
+
+::
+
+	from coalib.bearlib.abstractions.Lint import Lint
+	from coalib.bears.LocalBear import LocalBear
+
+	class HTMLTutorialBear(LocalBear, Lint):
+
+To make our bear use a linter we will have to overwrite some of the
+predefined values of the Lint class. Some of the most important are 
+``executable`` and ``output_regex``.
+
+We use ``executable`` to specify the linter executable. In our case it would be
+
+::
+
+	executable = 'html_lint.py'
+
+The ``output_regex`` is used to group parts of the output (such as ``lines``, 
+``columns``, ``severity`` and ``message``) so it can be used by the Lint
+class to yield Results. (more on communicating with the user :doc:`Writing Bears <../Writing_Bears>`)
+
+In order to figure out the ``output_regex`` we have to first see how the
+linter output looks. I will use this file as ``sample.html``
+
+::
+
+	<html>
+	  <body>
+	    <h1>Hello, world!</h1>
+	  </body>
+	</html>
+
+Now we can run
+
+::
+
+	$ html_lint.py sample.html
+	1:1: Info: Optional Tags: Omit optional tags (optional): You may remove the opening "html" tag.
+	2:3: Info: Optional Tags: Omit optional tags (optional): You may remove the opening "body" tag.
+	4:3: Info: Optional Tags: Omit optional tags (optional): You may remove the closing "body" tag.
+	5:1: Info: Optional Tags: Omit optional tags (optional): You may remove the closing "html" tag.
+
+Our ``output_regex`` should look like this
+
+::
+
+	(line):(column): (severity): (message)
+
+Or in `python regex <https://docs.python.org/2/library/re.html>`__
+
+::
+
+	(?P<line>\d+):(?P<column>\d+):\s(?P<severity>Error|Warning|Info):\s(?P<message>.+)
+
+Now that we found out our ``output_regex`` (we might want to test it just to be sure, I used 
```

Comment:
Maybe rephrase this to `we might want to test it just to be sure, regex101 is a great tool for such purposes`